timer_ract/src/App.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import "./App.css";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { timeLeft: null, interval: null, userInput: "" };
+  }
+
+  inputTime = () => {
+    if (new RegExp(/^\d+$/).test(this.state.userInput)) {
+      this.setState({ timeLeft: this.state.userInput });
+    } 
+  };
```

Comment:
Every time you write path to the attribute, consider using destructuring instead.
```suggestion
  inputTime = () => {
  const { userInput } = this.state;
    if (new RegExp(/^\d+$/).test(userInput)) {
      this.setState({ timeLeft: userInput });
    } 
  };
```